ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/DistributingExecutor.java

```diff
@@ -196,6 +203,13 @@ public StatementExecutorResponse execute(
           statement.getStatementText()), e);
     }
 
+    if (!rateLimiter.tryAcquire()) {
+      throw new KsqlRestException(
+        Errors.tooManyRequests(
+          "Too many writes to the command topic within a 1 second timeframe"
```

Comment:
nit: change this to "DDL/DML rate is crossing the configured rate limit of <rate-limit> statements/second"

Reply:
updated